docs/semgrep-code/notifications.md

```diff
@@ -402,40 +403,146 @@ If you are using GitHub Actions to run Semgrep, no extra changes are needed to g
-2. Create an API token in GitLab by going to [Profile > Access Tokens](https://gitlab.com/-/profile/personal_access_tokens) and adding a token with `api` scope.
-3. Copy the token created in the previous step.
-4. Navigate to **your repository >  Settings > CI/CD > Variables** and click **Expand**. The URL of the page where you are ends with: `/username/project/-/settings/ci_cd`.
-5. Click **Add variable**.
-6. Enter `PAT` for the `key` field and use the token you copied in step 3 as the value. 
-7. Select **mask variable** and unselect **protect variable**.
-8. Update your `.gitlab-ci.yml` file with variable `GITLAB_TOKEN` and value `$PAT`. Refer to the following example:
+1. In GitLab, go to [Profile > Access Tokens](https://gitlab.com/-/profile/personal_access_tokens), and then add a token with `api` scope.
+1. Copy the token created in the previous step.
+1. Navigate to **Your repository** >  **Settings** > **CI/CD**. The URL of the page where you are ends with: `/username/project/-/settings/ci_cd`.
+1. Under **Variables** click **Expand**, and then click **Add variable**.
+1. Enter **PAT** (change this placeholder name as necessary) in the **Key** field and paste the token value copied in step two to the **Value** field.
+1. Select the **Mask variable** checkbox option, and then clear the **Protect variable** checkbox option.
+1. Update your `.gitlab-ci.yml` file with variable `GITLAB_TOKEN` and value `$PAT`. Refer to the following example:
 ```yaml
 semgrep:
+  # A Docker image with Semgrep installed.
   image: returntocorp/semgrep
-  script:
-    - semgrep ci
+  # Run the "semgrep ci" command on the command line of the docker image.
+  script: semgrep ci
+
   rules:
+  # Scan changed files in MRs, (diff-aware scanning):
   - if: $CI_MERGE_REQUEST_IID
 
+  # Scan mainline (default) branches and report all findings.
+  - if: $CI_COMMIT_BRANCH == $CI_DEFAULT_BRANCH
+
   variables:
+    # Connect to Semgrep Cloud Platform through your SEMGREP_APP_TOKEN.
+    # Generate a token from Semgrep Cloud Platform > Settings
+    # and add it as a variable in your GitLab CI/CD project settings.
     SEMGREP_APP_TOKEN: $SEMGREP_APP_TOKEN
+    # Receive inline MR comments (requires Semgrep Cloud Platform account)
     GITLAB_TOKEN: $PAT
 ```
+Substitute the placeholder <code><span className="placeholder">PAT</span></code> with the name you created for this variable.
+
+For more configuration options, see [GitLab CI Sample](/semgrep-ci/sample-ci-configs/#gitlab-ci).
+
+:::info
+- Test Semgrep MR comments by submitting a test code from a rule in your [Rule board](https://semgrep.dev/orgs/-/board) that is in the **Comment** column.
+- Only rules in the **Comment** and **Block** columns of your [Rule board](https://semgrep.dev/orgs/-/board) create the MR comments. Rules from the **Block** will also block the MR pipeline. To unblock the pipeline, the detected code needs to be fixed.
```

Comment:
@s-santillan the same issue as you [mentioned below](https://github.com/returntocorp/semgrep-docs/pull/915/files#r1164532940), if you have a better way to test this, let's write it here. If not, let's delete this.
```suggestion
Only rules in the **Comment** and **Block** columns of your [Rule board](https://semgrep.dev/orgs/-/board) create the MR comments. Rules from the **Block** will also block the MR pipeline. To unblock the pipeline, the detected code needs to be fixed.
```